src/vs/workbench/contrib/inlineChat/browser/inlineChatWidget.ts

```diff
@@ -312,7 +312,7 @@ export class InlineChatWidget {
 	protected _doLayout(widgetDimension: Dimension, inputDimension: Dimension): void {
 		this._elements.progress.style.width = `${inputDimension.width}px`;
 		this._chatMessageContents.style.width = `${widgetDimension.width - 10}px`;
-		this._chatMessageContents.style.maxHeight = `270px`;
+		this._chatMessageContents.style.maxHeight = Math.min(270, widgetDimension.height) + 'px';
```

Comment:
We'll want to make sure this is handled well in the editor, try getting it to increase and reduce size to test

Reply:
What problem does this fix? I don't believe setting overall height as messages height is correct.

Reply:
The PR fixed the widget getting clipped when resized as it's in a fixed position on top of the terminal https://github.com/microsoft/vscode/issues/207601

Reply:
I believe there are bugs and this change might fix something but I don't believe this is the right fix. The max height of the chat contents cannot be the overall widget height.